sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/README.md

```diff
@@ -106,7 +106,6 @@ class Program {
 }
 
 class SimplePartitionProcessor extends PartitionProcessor {
-    @Override
```

Comment:
There's a javabug that prevents javadoc from correctly handling an @ symbol in a <code> however if instead of removing the line entirely it can be commented out. //@Override won't cause any issues with javadoc (I just verified)

Reply:
I've put override back in as commented out -- thanks Jim.